src/util/toggle-class.js

```diff
@@ -0,0 +1,38 @@
+//
+// helper to add/remove classes for IE9
+//
+// Usage:
+//
+// toggleClass(div, 'demo'); // Toggles the class `demo`
+// toggleClass(div, 'demo', true); // Adds the class `demo`
+// toggleClass(div, 'demo', false); // removes the class `demo`
+//
+export default function toggleClass(element, className, toggleClass) {
+  const classes = element.className === '' ? [] : element.className.split(' ');
+  const idx = classes.indexOf(className);
```

Comment:
while admittedly not very relevant, I have a strong distaste for abbreviations like that.